packages/core/src/external/opensearch/file-search-connector-factory.ts

```diff
@@ -46,7 +46,7 @@ export function makeSearchServiceQuery(): OpenSearchFileSearcher {
 
 export function makeSearchServiceRemover(): OpenSearchFileRemover {
   const region = Config.getAWSRegion();
-  const endpoint = Config.getSearchEndpoint();
+  const endpoint = "https://" + Config.getSearchEndpoint();
```

Comment:
@leite08 was getting `Error deleting from OpenSearch: Invalid URL (ERR_INVALID_URL)` i assume its because im missing this but let me know if thats incorrect

Reply:
Can check this on the configs, but I assume it's working otherwise no request would, right?

Reply:
We have it set as so in multiple places so assume it needs ot be there